tensorflow/lite/micro/kernels/reduce.cc

```diff
@@ -30,6 +31,16 @@ namespace reduce {
 constexpr int kMaxNumberOfAxis = 4;
 constexpr int kMaxNumberOfReducedAxis = 2;
 
+struct OpData {
+  int32_t multiplier;
+  int shift;
+};
+
+void* InitMean(TfLiteContext* context, const char* buffer, size_t length) {
+  auto* op_data = new OpData();
```

Comment:
void* raw;
  context->AllocatePersistentBuffer(context, sizeof(OpData), &raw);
  return raw;

Reply:
yes, please make this change. We can not have dynamic memory allocation for micro and the AllocatePersistentBuffer API is the way to go.

Reply:
fixed in latest commit